### ISSUE TITLE
Getting Started Guide

### ISSUE THREAD

**Issue by ForbesLindesay** [pugjs/pug#3036]
I should figure out incorporating https://itnext.io/pug-js-to-make-your-life-easier-with-html-templates-9c62273626e0 by @thatisuday into our official documentation at some point. It's an excellent in-depth getting started guide, and covers a lot more detail than our guide does.

**Comment by graham-sportsmgmt** [comment]
I'd be happy to write this for you in standard documentation style.  Do you have permission from @thatisuday to incorporate this work into the pug repository?

**Comment by ForbesLindesay** [comment]
No, I haven't got as far as asking yet. I just wanted to make this note before I forgot.

**Comment by graham-sportsmgmt** [comment]
I just asked him to post his approval here.

**Comment by thatisuday** [comment]
@ForbesLindesay @graham-sportsmgmt You have my permission. Go ahead.

**Comment by ForbesLindesay** [comment]
Thanks @thatisuday 

@graham-sportsmgmt would you still be happy to write up our getting started guide? The first step is to add a markdown file to https://github.com/pugjs/pug-en/tree/master/src then you can add a link to it in https://github.com/pugjs/pug-www/blob/3980e3cc3206c50a377878a9bae99b855744c5c8/templates/_menu.pug

If you clone both repos into the same parent directory, you should be able to run the website locally to preview your changes (in theory)

**Comment by graham-sportsmgmt** [comment]
Will do.

**Comment by graham-sportsmgmt** [comment]
Quick update - I'm working on this bit by bit locally.  It would a lot easier if github would let me save in increments :)
